resources/roca_lib.brs

```diff
@@ -391,6 +387,11 @@ sub __suite_exec(args as object)
     else
         tap.pass(args.index, description)
     end if
+
+    ' we may have changed context during execution, so let's update our parent's context
```

Comment:
This was a bug I discovered with context that was preventing tests from passing.

Reply:
Since context changes are now persisted between tests, do we still need the big scary warning in about "accessing `m.__suite` and `m.__suite.__ctx`?  I'd love to get rid of those things :)

Reply:
Great callout! Removed them 😄